website/content/api-docs/allocations.mdx

```diff
@@ -1039,5 +1039,22 @@ $ nomad operator api /v1/client/allocation/23f520cc-629a-46ff-395f-0661e7aa939e/
 }
 ```
 
+#### Field Reference
+
+- `ScheduledState` `(string)`: The task's current paused state. It can can have
```

Comment:
```suggestion
- `ScheduleState` `(string)`: The task's current paused state. It can can have
```

per https://github.com/hashicorp/nomad/blob/v1.9.0/api/allocations.go#L549-L551, which I now notice also has an inaccurate docstring :face_exhaling: